src/wazuh_mcp_server/auth.py

```
@@ -0,0 +1,355 @@
+        if api_keys_json:
+            try:
+                keys_data = json.loads(api_keys_json)
+                for key_data in keys_data:
+                    api_key = APIKey(**key_data)
+                    self.api_keys[api_key.id] = api_key
+            except Exception as e:
+                print(f"Error loading API keys: {e}")
+        
+        # Create default API key if none exist
+        if not self.api_keys:
+            default_key = self.create_api_key(
```

Ruff (0.12.2)

Local variable `default_key` is assigned to but never used

Remove assignment to unused variable `default_key`

(F841)